src/main/java/com/rtb/projectmanagementtool/project/ProjectData.java

```diff
@@ -236,16 +220,21 @@ public boolean equals(Object other) {
     return other instanceof ProjectData && equals(this, (ProjectData) other);
   }
 
-  /** @return the string representation of this class. */
+  /** @return the string representation of this class in JSON format. */
   @Override
   public String toString() {
-    String returnString = "{\n";
-    returnString += "Project id: " + this.id + "\n";
-    returnString += "Project creator's id: " + this.creatorId + "\n";
-    returnString += "Project Name: " + this.name + "\n";
-    returnString += "Project Description: " + this.description + "\n";
-    returnString += "Project Members: " + this.admins.toString() + "\n";
-    returnString += "Project Admins: " + this.members.toString() + "\n";
-    return returnString;
+    String json = "{";
+    json += "\"name\":";
+    json += "\"" + this.name + "\",";
+    json += "\"description\":";
+    json += "\"" + this.description + "\",";
+    json += "\"creatorId\":";
+    json += "\"" + this.creatorId + "\",";
+    json += "\"members\":";
+    json += "\"" + this.members.toString() + "\",";
+    json += "\"admins\":";
+    json += "\"" + this.admins.toString() + "\"";
+    json += "}";
+    return json;
   }
```

Comment:
I'm assuming you changed `toString()` to print ProjectData as JSON because that's how it'll be used in the servlet. Is there a reason for doing it like this manually instead of using Gson, like `gson.toJson(your_data)`?

Reply:
gson.toJson(ProjectData) includes the private variables of ProjectData in the resulting Json, and while it shouldn't affect functionality, it doesn't look good. Depending on how javascript reacts to this I might use gson.toJson

Reply:
That makes sense.

Reply:
If that's the case `toString` maybe shouldn't be used to send back to the JS as part of the response.  `toString` by convention is used to dump the whole class onto a string (eg. for debugging purposes) and would probably be ideal to also return private members.

If you need a version that does not include specific members for JS consumption, consider adding a "formatForResponse()" method or something similar